source/BcmI2cController.h

```diff
@@ -265,8 +265,8 @@ class BcmI2cControllerClass : public I2cControllerClass
     const ULONG _DIV_USED_MASK = 0x0000FFFF;  // Mask of non-reserved bits in the Clock Divider Register
 
     // Standard (100khz) and high (400khz) divider values.
-    const ULONG CDIV_100KHZ = 1500;
-    const ULONG CDIV_400KHZ = 376;      // Actually 398.9khz, divider value must be even
+    const ULONG CDIV_100KHZ = 2500;
```

Comment:
We definitely need a unit test here - should be part of the overall test plan.
It was tested before increasing the RPI2 clock speed. However, with the clock speed upgrade, both normal and high i2c bus speeds became wrong (too high).